src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```diff
@@ -1,480 +1,480 @@
+    "VIOLATION::CUSTOMER_EMAIL_NOT_UNIQUE": "Diese E-Mail-Adresse ist bereits registriert.",
+    "VIOLATION::CUSTOMER_PASSWORD_NOT_CORRECT": "Das Passwort ist nicht korrekt.",
+    "message-default": "Leider ist etwas schiefgelaufen.",
+    "message-404": "Die angeforderte Seite kann nicht gefunden werden.",
+    "addToCartError": "Beim Hinzufügen zum Warenkorb ist ein Fehler aufgetreten.",
+    "FRAMEWORK__INVALID_UUID": "Die gewählte Zahlungsart konnte nicht gefunden werden.",
+    "CHECKOUT__UNKNOWN_PAYMENT_METHOD": "Die gewählte Zahlungsart konnte nicht gefunden werden.",
+    "productNotFound": "Produkt \"%number%\" konnte nicht gefunden werden.",
+    "payment-method-blocked": "Die gewählte Zahlungsart %name% ist nicht verfügbar.",
+    "shipping-method-blocked": "Die gewählte Versandart %name% ist nicht verfügbar.",
+    "shipping-address-blocked": "Lieferungen an die gewählte Lieferadresse sind aktuell nicht möglich.",
+    "VIOLATION::TOO_SHORT_ERROR": "%field% ist zu kurz.",
+    "product-not-found": "Das Produkt wurde nicht gefunden.",
+    "message-403-ajax": "Deine Sitzung ist abgelaufen. Bitte lade die Seite neu und versuche es erneut.",
+    "message-403": "Deine Sitzung ist abgelaufen. Bitte lade die letzte Seite neu und versuche es erneut."
+  },
+  "account": {
+    "confirmationIsAlreadyDone": "Die Bestätigung ist bereits erfolgt oder die URL ist ungültig.",
+    "doubleOptInMailConfirmationSuccessfully": "Vielen Dank für die Bestätigung Ihrer E-Mail! Sie können Ihre Bestellung nun abschließen.",
+    "doubleOptInRegistrationSuccessfully": "Vielen Dank für Ihre Anmeldung! Ihr Account wurde erfolgreich freigeschaltet.",
+    "myAccount": "Mein Konto",
+    "registerTitle": "Registrierung",
+    "loginHeader": "Ich bin bereits Kunde",
+    "loginFormDescription": "Einloggen mit E-Mail-Adresse und Passwort",
+    "loginMailLabel": "Ihre E-Mail-Adresse",
+    "loginMailPlaceholder": "E-Mail-Adresse",
+    "loginPasswordLabel": "Ihr Passwort",
+    "loginPasswordPlaceholder": "Passwort",
+    "loginPasswordRecover": "Ich habe mein Passwort vergessen.",
+    "loginSubmit": "Anmelden",
+    "loginBadCredentials": "Es konnte kein Account zu den angegebenen Zugangsdaten gefunden werden.",
+    "registerAdvantagesHeader": "Meine Vorteile:",
+    "registerAdvantage1": "Schnelles Einkaufen",
+    "registerAdvantage2": "Speichern Sie Ihre Daten und Einstellungen",
+    "registerAdvantage3": "Einblick in Ihre Bestellungen inkl. Versandinformationen",
+    "registerAdvantage4": "Verwalten Sie ihr Newsletter-Abonnement",
+    "registerPersonalHeader": "Ich bin Neukunde",
+    "registerAddressBillingHeader": "Ihre Adresse",
+    "registerAddressShippingHeader": "Ihre abweichende Lieferadresse",
+    "personalTypeLabel": "Ich bin",
+    "personalTypePrivate": "Privat",
+    "personalTypeBusiness": "Gewerblich",
+    "personalSalutationLabel": "Anrede",
+    "personalSalutationPlaceholder": "Anrede eingeben...",
```

Comment:
Missing spaces before "..." (I'll comment it just once, because u'll search+replac anyway)